src/ReactiveSocket.cpp

```diff
@@ -259,11 +268,26 @@ bool ReactiveSocket::createResponder(
   return true;
 }
 
+bool ReactiveSocket::resumeListener(const ResumeIdentificationToken& token) {
+    return resumeSocketListener_(*this, token);
+}
+
 void ReactiveSocket::close() {
   connection_->disconnect();
 }
 
 void ReactiveSocket::onClose(CloseListener listener) {
   connection_->onClose([listener, this]() { listener(*this); });
 }
+
+void ReactiveSocket::resumeFromSocket(ReactiveSocket &socket) {
+  connection_->resumeFromAutomaton(*socket.connection_);
+}
+
+void ReactiveSocket::tryClientResume(
```

Comment:
since this is tryXXX method, how do we let the caller know whether this succeeded or failed?